docs/installLiberty.md

```diff
@@ -5,11 +5,39 @@ The `installLiberty` task is used to download and install WebSphere Liberty serv
 The task can download the WebSphere Liberty runtime archive in three ways: 
 * from a specified location (via `runtimeUrl`)
 * from the [Liberty repository](https://developer.ibm.com/wasdev/downloads/) based on a version and a runtime type
-* from the Maven Central repository.  
+* from the [Maven Central repository](http://search.maven.org/) using the dependencies block.  
 
 When installing Liberty from a JAR file, the Liberty license code is needed to install the runtime. When you are installing Liberty from the Liberty repository, you can see the versions of Liberty available to install and find the link to their license using the index.yml file. After opening the license, look for the `D/N: <license code>` line. Otherwise, download the runtime archive and execute `java -jar wlp*runtime.jar --viewLicenseInfo` command and look for the `D/N: <license code>` line.
```

Comment:
....find the link to their license using the [index.yml](https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/downloads/wlp/index.yml) file.